docs/js/scripts.js

```diff
@@ -0,0 +1,59 @@
+function setMembersList(membersString) {
+	document.getElementById("members").innerHTML = membersString;
+}
+
+function setProjectsList(nodes) {
+	var list = document.getElementById("projects");
+	for(node of nodes) {
+		list.appendChild(node);
+	}
+}
+
+function getProjectsFromData() {
```

Comment:
This and the below function are very similar, should we turn them into one function with some parameters?